setting-up-webauthn-mfa.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Setting up WebAuthn / Passkey multi-factor authentication
+url: /setting-up-webauthn-mfa
+previous: /setting-up-multifactor-authentication
+next: /setting-up-otp-mfa
+---
+
+<em class="t-gray">Setting up WebAuthn multifactor authentication</em>
+
+Using WebAuthn for multifactor authentication (MFA) is the best way to
+protect your account from takeover. It's stronger and easier to use than
+[OTP codes](/setting-up-otp-mfa).
+
+## Prerequisite
+
+To use WebAuthn, you will need at least _one_ of the following:
+
+* A hardware security token (sometimes called a security key), such as a
+  YubiKey or Google Titan Key.
+* A built-in hardware device, such as TouchID, FaceID or Windows Hello.
+* A browser that supports the "Passkey" standard. Up-to-date versions of
+  Chrome, Safari, Firefox and Edge all support this standard.
+
+Unfortunately implementations of these experiences vary, so we can't show
+the exact details, but we will point out the steps that are specific to
+using rubygems.org.
+
+## Enabling WebAuthn multifactor authentication
+
+1. Login to rubygems.org using your existing account and go to the
+[edit settings](https://rubygems.org/settings/edit) page.
+2. In the _multifactor authentication_ section you will see two options:
+"Authentication App" and "Security Device". Under "Security Device" you
+will see a field for Nickname.
+  ![Nickname for security device on the edit settings page](/images/enabling_webauthn_nickname.png){:class="t-img"}
+3. Choose a name for your device. Use something that helps you remember
+which device you used. For example, you might use nicknames like "Mary's
+YubiKey" or "Naveen's iPhone".
+4. Below the Nickname field, click **Register device**.
```

Comment:
<img width="789" alt="Screenshot 2023-03-28 at 4 27 18 PM" src="https://user-images.githubusercontent.com/42748004/228358942-7d116dd8-8a7a-4703-bf12-8298276130f3.png">

I think there's too many styles to signify an example / section of the site (italicizing, bold, "s). If I had to choose one, I'd prefer the Bold since it pops more.

Reply:
I followed the style from the original OTP page. I'll change both. Elsewhere in the docs it uses "Quotes for Page Titles" and **Bold for Buttons**, so I'll adopt that and update them both.